src/ORM/FieldType/DBComposite.php

```diff
@@ -8,7 +8,7 @@
 use SilverStripe\ORM\Queries\SQLSelect;
 
 /**
- * Apply this interface to any {@link DBField} that doesn't have a 1-1 mapping with a database field.
+ * Use this template when subclassing any {@link DBField} that doesn't have a 1-1 mapping with a database field.
```

Comment:
```suggestion
 * Extend this class when designing a {@link DBField} that doesn't have a 1-1 mapping with a database field.
```

Reply:
Makes sense to me interface just threw me off as a newer developer. I'd imagine etiquette is to squash/fixup (if I got those terms right) these two commits but this is as far as time will let me take this PR. Sorry!